packages/grafbase-sdk/src/resolver/context.ts

```diff
@@ -0,0 +1,68 @@
+import { KVListResult, KVMetadata } from './context/kv'
+import { Classification } from './context/ai'
+
+/**
+ * The type of the `context` argument in a Grafbase edge resolver.
+ *
+ * Reference: https://grafbase.com/docs/edge-gateway/resolvers#context
+ *
+ * @example
+ *
+ * import { Context, Info } from '@grafbase/sdk'
+ *
+ * export default async function(_parent, _args, context: Context) {
+ *   // ...
+ * }
+ */
+export type ResolverContext = {
+  /** Context about the HTTP request being handled. */
+  request: {
+    headers: Record<string, any>,
+  }
+  /**
+   * Grafbase KV
+   *
+   * If you want to use this, please make sure to [enable KV in your Grafbase configuration](https://grafbase.com/docs/edge-gateway/resolvers#enable-kv).
+   *
+   * See the reference documentation: https://grafbase.com/docs/edge-gateway/resolvers#kv
+   */
+  kv: {
```

Comment:
There's also [`delete`](https://grafbase.com/docs/edge-gateway/resolvers#delete)

Reply:
Should be fixed now.